cypress/integration/manageLayers.cy.js

```diff
@@ -0,0 +1,198 @@
+import { EXTENDED_TIMEOUT, getApiBaseUrl } from '../support/util.js'
+
+const MAPS_ADMIN_AUTHORITY_ID = 'F_PROGRAM_TRACKED_ENTITY_ATTRIBUTE_GROUP_ADD'
```

Comment:
is this the correct authority?

Reply:
chore: [da2968a](https://github.com/dhis2/maps-app/pull/3198/commits/da2968ad903b878ab66a55bdd6b8731bf6d69fc8)